### ISSUE TITLE
flip-up HUD prop should have the display transform as a child of the glass

### ISSUE THREAD

**Issue by JonnyOThan** [StoneBlue/ASET-Consolidated-Avionics#2]
Right now if you turn on the display as the animation is playing, it's very obvious that the readout isn't connected to the glass itself.  I don't think there's any reason why just changing the hierarchy wouldn't work.

**Comment by StoneBlue** [comment]
Hmm.. I'ld probably have to adjust that prop *only* the new one in MAS (NoBody), doesnt *have* glass..

**Comment by JonnyOThan** [comment]
I'm not quite sure what you mean here.  I believe this could probably be done by just putting whatever transform actually contains the dynamic content as a child of the glass object (which moves).

**Comment by StoneBlue** [comment]
Yeah.. I just looked.. there is a completely seperate model in MAS itself, that is just a duplicate of, and contains *only* the invisible screen plane and the display transforms *only*. It also uses seperate switch props to activate/scale/move/brightness/color it.
It *does* need/reference the textures in ASET Avionics, so as long as those arent being changed.

So yeah, no overalp with the ASET Avionics version.
Sorry... mainly just commented as a reminder for me to check & make sure there would be no conflict changing the original in ASET. :P
